oxd-server/src/main/java/org/gluu/oxd/server/op/GetRequestUriOperation.java

```diff
@@ -0,0 +1,121 @@
+package org.gluu.oxd.server.op;
+
+import com.google.inject.Injector;
+import io.dropwizard.util.Strings;
+import org.apache.commons.lang.StringUtils;
+import org.gluu.oxauth.model.crypto.signature.SignatureAlgorithm;
+import org.gluu.oxauth.model.jwk.Algorithm;
+import org.gluu.oxauth.model.jwk.Use;
+import org.gluu.oxauth.model.jwt.Jwt;
+import org.gluu.oxauth.model.jwt.JwtType;
+import org.gluu.oxd.common.Command;
+import org.gluu.oxd.common.ErrorResponseCode;
+import org.gluu.oxd.common.params.GetRequestUriParams;
+import org.gluu.oxd.common.response.GetRequestUriResponse;
+import org.gluu.oxd.common.response.IOpResponse;
+import org.gluu.oxd.server.HttpException;
+import org.gluu.oxd.server.Utils;
+import org.gluu.oxd.server.service.Rp;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+public class GetRequestUriOperation extends BaseOperation<GetRequestUriParams> {
```

Comment:
GetRequestObjectUriOperation (same for Params)

Reply:
Done